docker/stellargraph-treon/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Image is needed anyway, so it's also used for downloading datasets
+FROM python:3.6-slim as base
+
+WORKDIR /data
+
+# Download datasets
+# hadolint ignore=DL3008
+RUN apt-get update \
+      && apt-get install --no-install-recommends -y curl unzip \
+      && curl -sLO "https://linqs-data.soe.ucsc.edu/public/lbc/cora.tgz" \
+      && tar --no-same-owner -xf cora.tgz \
+      && curl -sLO "https://linqs-data.soe.ucsc.edu/public/Pubmed-Diabetes.tgz" \
+      && tar --no-same-owner -xf Pubmed-Diabetes.tgz \
+     # && curl -sLO "http://files.grouplens.org/datasets/movielens/ml-100k.zip" \
+     # && unzip ml-100k.zip \
+      && curl -sLO "http://socialcomputing.asu.edu/uploads/1283153973/BlogCatalog-dataset.zip" \
+      && unzip BlogCatalog-dataset.zip
+
+# hadolint ignore=DL3006
+FROM stellargraph/stellargraph
+
+ENV JUPYTER_VERSION="1.0.0" \
+  PATH=${PATH}:/home/stellar/.local/bin
+  
+ENV TREON_VERSION="0.1.2" \
```

Comment:
These can probably be merged in one `ENV`, e.g.,

```
ENV PATH=${PATH}:/home/stellar/.local/bin \
  JUPYTER_VERSION="1.0.0" \
  TREON_VERSION="0.1.2" \
  SEABORN_VERSION="0.9.0"
```

Reply:
Merged all the version instructions in one command.